syne_tune/optimizer/transfer_learning/bounding_box.py

```diff
@@ -0,0 +1,116 @@
+import logging
+from typing import Dict, Callable, Optional
+import pandas as pd
+
+import syne_tune.search_space as sp
+from syne_tune.optimizer.scheduler import TrialScheduler
+from syne_tune.optimizer.transfer_learning import TransferLearningScheduler, TransferLearningTaskEvaluations
+from syne_tune.search_space import Categorical
+
+
+class BoundingBox(TransferLearningScheduler):
```

Comment:
Just a suggestion: Would it not be cleaner to make this a scheduler factory in a way? That would not need the lambda function then, and also not this self.scheduler and replicating all the methods, which is not so elegant.